lib/workers/repository/onboarding/pr/config-description.ts

```diff
@@ -52,9 +52,13 @@ export function getConfigDesc(
   });
   desc += '\n';
   desc += emojify(
-    `:abcd: Would you like to change the way Renovate is upgrading your dependencies?`,
+    `:abcd: Do you want to change how Renovate upgrades your dependencies?`,
```

Comment:
I prefet the existing wording.

Reply:
I applied thisesuggestion https://github.com/renovatebot/renovate/pull/25899#discussion_r1401801560.
I can for sure revert it if you want me to

Reply:
It's okay if Honking Goose suggested it. He is better at this 👍